rebar.config.script

```diff
@@ -1,6 +1,5 @@
-IsRebar3 = erlang:function_exported(rebar3, main, 1),
 
```

Comment:
Please remove the stray empty lines at the top and bottom of this file, and prepend an entry describing this change in the top-level `changelog.md`, with tag `2.9.8`.